database/wireless_regdb.ksy

```diff
@@ -0,0 +1,202 @@
+meta:
+  id: wireless_regdb
+  title: Linux wireless regulation database (aka Central Regulatory (Domain Agent) DataBase)
+  license: copyleft-next-0.3.0 AND ISC
+  endian: be
+  bit-endian: be
+  encoding: ASCII
+
+-license-URIs:
+  - https://github.com/copyleft-next/copyleft-next/blob/master/Releases/copyleft-next-0.3.0
```

Comment:
This is not necessary. The SPDX expression `copyleft-next-0.3.0 AND ISC` specified in `meta/license` unambiguously identifies the license on its own. If anyone wants to read the full text of the license, it's always available at `https://spdx.org/licenses/{$identifier}.html` (e.g. [https://spdx.org/licenses/`copyleft-next-0.3.0`.html](https://spdx.org/licenses/copyleft-next-0.3.0.html)).

And BTW, the key `-license-URIs` does not really follow any reputable naming convention (sure, no strict rules apply for dash-prefixed keys, but you know: some sort of consistency is always beneficial). But I'd rather remove it completely.